rewrite-core/src/main/java/org/openrewrite/Result.java

```diff
@@ -185,13 +185,23 @@ public String diff() {
      * @return Git-style patch diff representing the changes to this compilation unit.
      */
     public String diff(@Nullable Path relativeTo) {
-        return diff(relativeTo, null);
+        return diff(relativeTo, (PrintOutputCapture.MarkerPrinter) null);
     }
 
     public String diff(@Nullable Path relativeTo, PrintOutputCapture.@Nullable MarkerPrinter markerPrinter) {
         return diff(relativeTo, markerPrinter, false);
     }
 
+    @Incubating(since = "8.41.4")
```

Comment:
If we move this `StandardMarkerPrinters` then we don't need to add these overloads. Indeed I would prefer we don't add more overload variations here unless absolutely necessary. Every new overload is one more item in the IDE autocomplete list.